src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -15,17 +14,17 @@ export interface DescriptionListProps {
   items: Item[];
 }
 
-const getUniqueTermKey = createUniqueIDFactory(`Term`);
-const getUniqueDescriptionKey = createUniqueIDFactory(`Description`);
-
 export function DescriptionList({items}: DescriptionListProps) {
+  // There's no good key to give React so using the index is a last resport.
+  // we can't use the term/description value as it may be a react component
+  // which can't be stringified
```

Comment:
Why not generate unique IDs here too?

Reply:
keys only need to be unique within the current loop so there's no extra value gained by using a globally unique value instead of a locally unique value (i.e. the loop index)